pyeudiw/federation/schemas/wallet_relying_party.py

```diff
@@ -1,45 +1,51 @@
-from typing import Any, Dict, List
+from enum import Enum
```

Comment:
we need to distiguish th entity configuration in federation and the RP metadata in openid4vp

all about openid4vp must be movedi n the openid4vp package and the EC in federation must use the metadata defined there

this allow us to use rp metadata even outside federation

Reply:
I moved the the code relative to VP to the relative package in commit 98d494d.
It's ok?

Reply:
wallet_relying_party is a federation name

please, use a name agnostic to the trust model, wallet_relying_party.py -> __init__.py if it is related to openid4vp.schemas

Reply:
Resolved in [33d8000](https://github.com/italia/eudi-wallet-it-python/pull/138/commits/33d80005e1d86e8f613dc60f979c7af127aa9d1c)